pkgs/jnigen/lib/src/bindings/dart_generator.dart

```diff
@@ -2,6 +2,7 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:ffi';
```

Comment:
This seems to be an analyzer bug! We have `unused_import` lint, and we don't use `dart:ffi` anywhere!

Reply:
Report it!